src/main/java/servlets/NewCommentServlet.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 Google LLC
```

Comment:
Why did you make a seperate servlet for this? This should be combined with DisplayCommentsServlet, to just be "CommentServlet". When you want to create a new comment, send a post request to it. When you want to retrieve comments, make a get request.

Reply:
What difference does it make if its combined or not combined..? As in why the "should"? If you look at the week 4 walkthrough examples - namely to-do list - the two commands (doGet and doPost) are kept in separate files, and I prefer it because it makes very clear what each servlet is doing.

Reply:
Discussed offline.

Reply:
Please add results of discussions to such comment threads for posterity.

Reply:
Sorry about that - the result is that we decided the separation is alright in this case, because there are no strict guidelines enforcing combination, and the author prefers it in this case because the different links - /new-comment and /display-comments - makes it clearer at each stage what is happening. This also appears to be what is happening (separation making things clearer, since doPost only creates an entry to store in datastore, and doGet only fetches entries from datastore) in the to-do list example in Week 3, which sets it apart from other examples that do combine the two servlets.

Reply:
SG. Thanks.